src/lib.rs

```diff
@@ -146,6 +146,15 @@ pub mod socket_embedded_nal_tcp;
 #[cfg(riot_module_periph_gpio)]
 pub mod gpio;
 
+#[cfg(riot_module_periph_hwrng)]
+pub mod hwrng;
```

Comment:
As I understand, the sensible way to use all of this is through the prng module.

Is there any good reason that the hwrng module and the random module are pub in the first place, or that RandomSeed::new_from_hwrng would be public?

Reply:
Yes it is intended to be used through `prng`. I just left the rest public in case someone might need it e.g to seed his own prng but we could make it private to make people use the "intended" way. I do not have a strong opinion on that matter.

Reply:
I made the two modules `random` and `hwrng` private